CHANGELOG.rst

```diff
@@ -24,6 +24,8 @@ Changelog
 * Add Maestro credit card. Thanks @anthonylauzon.
 * Add ``hr_HR`` localization. Thanks @mislavcimpersak.
 * Update ``de_DE`` first names. Thanks @WarrenFaith and @mschoebel.
+* Allow generation of IPv4 and IPv6 network address with valid CIDR.
+* Unittest IPv4 and IPv6 address and network generation.
```

Comment:
thank you, very much appreciated! Feel free to add your name next to those entries :)